omnibus/files/private-chef-cookbooks/private-chef/templates/default/openssl.cnf.erb

```diff
@@ -0,0 +1,361 @@
+#
+# OpenSSL example configuration file.
```

Comment:
[nit] I'd be a little disappointed finding that comment in the rendered template :/ But that's just me